### PR TITLE
Add docs for YKD_SERIALISE_COMPILATION.

### PR DESCRIPTION
**Pull request by vext01**

### CODE REVIEW

File: src/dev/env.md [hunk 1]
```diff
@@ -19,6 +19,15 @@ The following stages are supported:
  - `jit-post-opt`: the IR for the trace after LLVM has optimised it. This is
    the IR that will be submitted to the LLVM code generator.
 
+### `YKD_SERIALISE_COMPILATION`
+
+If `YKD_SERIALISE_COMPILATION=1`, when an interpreter thread calls the the
```
Comment (ltratt) [inline src/dev/env.md:24]: This sentence doesn't really parse. Rephrase?

Comment (vext01) [reply src/dev/env.md]: Does 593b49c help?

**Comment by vext01** [comment]
Fixed in 2c17bb9

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Pull request merged**